### DISCUSSION TITLE
Note のクロールが 10月から停まっていますか？

### DISCUSSION THREAD

**Discussion by ykws** [lapras-inc/lapras-user-community#28] [Q&A] [answered]
2024年9月まではクロールされています。
2024年10月以降の記事はクロールされなくなっているように自分のアカウントからは見えています。

Note の連携が終了している可能性も考えられ、質問しようと思っていたところ、ちょうど Note を TechArticle に追加する discussion が登録されているので、 2024年12月現在もクロール対象なのは変わりないでしょうか

- #27

**Comment by naga3** [comment, marked as answer]
@ykws 

お知らせありがとうございます。

弊社のnoteクローラー不具合により、ykws様のクロール状況が2024/09/15が最終クロール日になっておりました。
現在は修正し、最新の記事までクロールされております。お手数をおかけしました。

> 2024年12月現在もクロール対象なのは変わりないでしょうか

はい、現在もクロール対象です。

また何かおかしい動きがございましたらご連絡ください。

今回はお手数をおかけして大変申し訳ありませんでした。

**Reply by ykws** [reply]
@naga3 早速調査いただきありがとうございます！
反映されていることを確認できました！
